src/main/java/com/todolist/enterprise/service/TodoServiceStub.java

```diff
@@ -0,0 +1,19 @@
+package com.todolist.enterprise.service;
+import org.springframework.stereotype.Component;
+import com.todolist.enterprise.dto.Task;
+import java.util.Date;
+
+@Component
+public class TodoServiceStub implements ITodoService {
+    @Override
+    public Task getTaskById(int taskId) {
+        if (taskId == 1) {
+            Task task = new Task();
+            task.setName("Task 1");
+            task.setDueDate(new Date());
+            task.setCompleted(false);
+            task.setListId(1);
```

Comment:
```suggestion
            task.setListId(1);
            return task;
```
I'm assuming since this is a get request, you probably want to return the task you created here.